docs/usage/jssprovider.md

```diff
@@ -0,0 +1,205 @@
+# `Mozilla-JSS` Provider
+
+The `Mozilla-JSS` JCA-compatible Provider exposes most of the functionality
+of JSS to external packages. This interface is the recommend interface most
+developers should build against. However, once the dependencies are satisfied
+and JSS's native component are available to the JVM, we still have to load
+and initialize JSS.
+
+There are two routes to do this:
+
+ 1. Via the `CryptoManager` interface, _and_
+ 2. Via `java.security`, directly loading the `JSSProvider`.
+
+
+## Loading JSS via `CryptoManager`
+
+To load JSS from a `CryptoManager`, it is necessary to decide what level of
+configuration is necessary. If you're happy with the defaults, it is
+sufficient to only specify a NSS DB:
+
+```java
+import org.mozilla.jss.CryptoManager;
+
+CryptoManager.initialize("/path/to/nss-db");
+```
+
+At this point, JSS will be initialized and can be used. However, a password
+might still be required, so see the section below for providing password
+callback handlers.
+
+However, certain default values might not work in all situations. For
```

Comment:
How about removing the "however" here since the previous paragraph also contains a "however"?